README.md

```diff
@@ -85,6 +86,7 @@ A curated list of anime streaming/downloading websites.
 
  * [animebytes.tv](https://animebytes.tv/)
  * [animetorrents.me](https://animetorrents.me/)
+ * [bakabt.me](https://bakabt.me)
```

Comment:
Also please explain how someone can get an account here :)

Reply:
If you want to get a bakabt account, you should go to their irc which is #BakaBT on Rizon, and take a queue number for getting reviewed.

Reply:
can you just add this message besides the link, see how others are done :)

Reply:
i suppose its okay now